guides/v2.3/graphql/reference/quote.md

```diff
@@ -24,11 +24,13 @@ The `Cart` object can contain the following attributes:
 
 Attribute |  Data Type | Description
 --- | --- | ---
-`applied_coupon` | code | Contains the coupon code if used 
+`applied_coupon` | code | Contains the coupon code if used
```

Comment:
The data type is now `AppliedCoupon`. Also update the description to say something like "The `AppliedCoupon` object contains the `code` text attribute, which provides the coupon code.